packages/package-store/src/storeController/index.ts

```diff
@@ -98,6 +104,69 @@ export default async function (
     }
   }
 
+  async function findPackageUsages (
+    dependencies: WantedDependency[]
+  ): Promise<FindPackageUsagesResponse[]> {
+
+    /**
+     * If `dependency` has no `alias`, then directly check against `pref`.
+     * This is messy due to overloaded use WantedDependency (npm, github, tarball, etc.)
+     * @param dependency the dependency to parse
+     */
+    function parseDependency (dependency: WantedDependency) {
+      let packageName;
+      let packageTag;
+      if (dependency.alias) {
+        packageName = `/${dependency.alias}/`;
```

Comment:
Because the user may specify a package without a version number, I can't just resolve the package and get a `packageId` to lookup in the `store`. Instead, I have to manually traverse the entire store and run substrings on every `packageId`.

This is inefficient, but that's OK. However, this breaks on Windows, as the store stores packages with `\` instead of `/`. Any reasonable way to fix this?

Reply:
I guess we could save it with `/` on Windows as well

Reply:
I was wrong about this, this actually does work in most cases. The `packageId` is resolved via an external service (npm, github), so it will always have `/` instead of `\` in it. Windows vs Unix shouldn't make a huge difference.

I believe it's only a problem when a file or tarball dependency is added to the store. I have not tested this yet, but I think this could be fixed in a separate PR since the majority of users use npm. I can create an issue for this tomorrow.